openslides/users/apps.py

```diff
@@ -29,5 +29,5 @@ def ready(self):
         register_slide_model(User, 'participant/user_slide.html')
 
         # Register viewsets.
-        router.register('users/user', UserViewSet)
-        router.register('users/group', GroupViewSet)
+        router.register('user/user', UserViewSet)
```

Comment:
I like it too - with plural names if it make sense (like we use it in the GUI). That means:
agenda, motions, assignments, users, files and of course one core!